Allow LIB_INSTALL_DIR to be set by the the cmake configure invocation.  We dont
easily have a way of determining something that patches ${LIB_SUFFIX} so we'll
set LIB_INSTALL_DIR.

Upstream-Status: Inappropriate [configuration]

Signed-off-by: Kumar Gala <galak@kernel.crashing.org>

Index: libmusicbrainz-3.0.3/CMakeLists.txt
===================================================================
--- libmusicbrainz-3.0.3.orig/CMakeLists.txt
+++ libmusicbrainz-3.0.3/CMakeLists.txt
@@ -26,7 +26,7 @@ FIND_PACKAGE(DiscId)
 SET(LIB_SUFFIX "" CACHE STRING "Define suffix of directory name (32/64)")
 SET(EXEC_INSTALL_PREFIX ${CMAKE_INSTALL_PREFIX} CACHE PATH "Installation prefix for executables and object code libraries" FORCE)
 SET(BIN_INSTALL_DIR ${EXEC_INSTALL_PREFIX}/bin CACHE PATH "Installation prefix for user executables" FORCE)
-SET(LIB_INSTALL_DIR ${EXEC_INSTALL_PREFIX}/lib${LIB_SUFFIX} CACHE PATH  "Installation prefix for object code libraries" FORCE)
+SET(LIB_INSTALL_DIR ${EXEC_INSTALL_PREFIX}/lib${LIB_SUFFIX} CACHE PATH  "Installation prefix for object code libraries")
 SET(INCLUDE_INSTALL_DIR ${CMAKE_INSTALL_PREFIX}/include CACHE PATH "Installation prefix for C header files" FORCE)
 
 CONFIGURE_FILE(${CMAKE_CURRENT_SOURCE_DIR}/libmusicbrainz3.pc.cmake ${CMAKE_CURRENT_BINARY_DIR}/libmusicbrainz3.pc)
